### DISCUSSION TITLE
你牛逼

### DISCUSSION THREAD

**Discussion by qinmaye** [upptime/upptime#159] [Q&A] [answered]
你又在搞一些我看不懂的东西了

**Comment by AnandChowdhary** [comment, marked as answer]
> 你又在搞一些我看不懂的东西了

[Google Translate says](https://translate.google.com/?sl=zh-CN&tl=en&text=你又在搞一些我看不懂的东西了%0A&op=translate):

> You're doing something I don't understand again

RTFM! 😉
